examples/community/go-jetpack/jetpack.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"fmt"
+	"image"
+	"io/ioutil"
+	"os"
+
+	_ "image/png"
+
+	"github.com/faiface/pixel"
+	"github.com/faiface/pixel/pixelgl"
+	"github.com/faiface/pixel/text"
+	"github.com/golang/freetype/truetype"
+	"golang.org/x/image/colornames"
+)
+
+func loadPicture(path string) (pixel.Picture, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+	img, _, err := image.Decode(file)
+	if err != nil {
+		return nil, err
+	}
+	return pixel.PictureDataFromImage(img), nil
+}
+
+func loadSprite(path string) (pixel.Sprite, error) {
+	pic, err := loadPicture(path)
+	if err != nil {
+		return *pixel.NewSprite(pic, pic.Bounds()), err
+	}
+	sprite := pixel.NewSprite(pic, pic.Bounds())
+	return *sprite, nil
+}
+
+func loadTTF(path string, size float64, origin pixel.Vec) *text.Text {
+	file, err := os.Open(path)
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	bytes, err := ioutil.ReadAll(file)
+	if err != nil {
+		panic(err)
+	}
+
+	font, err := truetype.Parse(bytes)
+	if err != nil {
+		panic(err)
+	}
+
+	face := truetype.NewFace(font, &truetype.Options{
+		Size:              size,
+		GlyphCacheEntries: 1,
+	})
+
+	atlas := text.NewAtlas(face, text.ASCII)
+
+	txt := text.New(origin, atlas)
+
+	return txt
+
+}
+
+func run() {
+	// Set up window configs
+	cfg := pixelgl.WindowConfig{ // Default: 1024 x 768
+		Title:  "Golang Jetpack!",
+		Bounds: pixel.R(0, 0, 1024, 768),
+		VSync:  true,
+	}
+	win, err := pixelgl.NewWindow(cfg)
+	if err != nil {
+		panic(err)
+	}
+
+	// Importantr variables
+	var jetX, jetY, velX, velY, radians float64 = 0, 0, 0, 0, 0
```

Comment:
you don’t need to assign these variables to zero, it’s default value is zero (https://play.golang.org/p/qYENMZMRA3S)